src/components/details/_macro-options.md

```diff
@@ -10,3 +10,5 @@
 | attributes        | object  | false    | HTML attributes (for example, data attributes) to add to the details                                                |
 | headingAttributes | object  | false    | HTML attributes (for example, data attributes) to add to the details header element                                 |
 | contentAttributes | object  | false    | HTML attributes (for example, data attributes) to add to the details content element                                |
+| isAccordion       | boolean | false    | Checks if the functionaltiy needs to be accordion. If true, adds the appropriate class                              |
+| group             | string  | false    | Used for accordian component where mutilple details components are grouped                                          |
```

Comment:
I think these two are only used through the accordion component and so do not need to be listed here. Because you wouldn't ever need to use them when using the details component on its own

Reply:
okay got it, so only params that I need to list are the ones that are independent of other components. Can I close this PR and ticket if changes are not required?

Reply:
Yeah so the user only needs to know about params they would use and pass in themselves. These two are ones that are passed in from the accordion component but the user wouldn't pass them in to the details component. So yeah this can be closed.